content/lessons/chapter-2/mining-1.tsx

```diff
@@ -187,53 +192,100 @@ export default function Mining1({ lang }) {
+              <BlockCounter blocks={blocks} total={1000} className="ml-auto" />
+            </div>
+            <ProgressBar progress={blocks / 10} />
+          </Card>
+          <Card className="flex" highlight={nonceHighlight}>
+            <div className="flex-1">
+              <TitleCard
+                title={t('chapter_two.mining_one.progress_bar_one')}
+                disabled={nonce === 0}
+              />
+              <NonceCounter
+                content={nonce}
+                disabled={nonce === 0}
+              ></NonceCounter>
+            </div>
+            <HashDisplayer
+              content={nonce}
+              disabled={nonce === 0}
+              step={step}
+              finalHash={
+                '000000000072947e2f22250fac0ddd882fcbf37cf6e2340a41129b6r23a2823a'
               }
-            )}
-          >
-            <span className={clsx({ 'fade-in text-[#EDA081]': blocks !== 0 })}>
-              {t('chapter_two.mining_one.progress_bar_title')}
-            </span>{' '}
-            <span className="absolute right-0">
-              {Intl.NumberFormat().format(blocks)} of 1,000
-            </span>
+              blockFound={blocks}
+            ></HashDisplayer>
+          </Card>
+          <Card className="flex-row" highlight={hashPowerHighlight}>
+            <div className="flex">
+              <div className="flex-1">
+                <TitleCard
+                  title={t('chapter_two.mining_one.progress_bar_two')}
+                  disabled={nonce === 0}
+                />
+                <HashFrequency
+                  disabled={nonce === 0}
+                  step={step}
+                  hashPower={hashPower}
+                />
+              </div>
+              <StartButton
+                hashPower={hashPower}
+                setHashPower={setHashPower}
+                onButtonClick={turnOnButton}
+                step={step}
+              ></StartButton>
+            </div>
+            <ProgressBar progress={hashPower / 440} variant={'bars'} />{' '}
+            {/* progress = hashPower * 100/maxHashPower */}
+          </Card>
+          <div className="flex items-center justify-between gap-x-2.5 text-center font-space-mono">
+            <Card className="w-max px-3.5 py-2.5">
+              <div
+                className={clsx('font-space-mono text-2xl', {
+                  'text-black/25': transactionsConfirmed === 0,
+                  'fade-in text-white': transactionsConfirmed !== 0,
+                })}
+              >
+                {Intl.NumberFormat().format(transactionsConfirmed)}
+              </div>
+              <TitleCard
+                title={t('chapter_two.mining_one.progress_bar_three')}
+                disabled={transactionsConfirmed === 0}
+                size={'small'}
+              />
+            </Card>
+            <Card className="w-max px-10 py-2.5">
+              <div
+                className={clsx('text-2xl font-normal', {
```

Comment:
this needs to be `font-space-mono`

Reply:
Addressed the highlight in https://github.com/saving-satoshi/saving-satoshi/commit/f2792a38f471fb6a75da4b2dca348ada7cac9aab

Thanks for catching it!